package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/answers-headless",
-  "version": "1.1.1-alpha.0-95",
+  "version": "1.1.1-alpha.0-96",
```

Comment:
I think we can remove the `0-` and just have `1.1.1-alpha.95` to match our other libraries

Reply:
updated!